src/baskerville/layers.py

```diff
@@ -23,11 +23,287 @@
 for device in gpu_devices:
     tf.config.experimental.set_memory_growth(device, True)
 
+#####################
```

Comment:
consider adding to transfer.py or an adapters.py file.

Reply:
done.